openstack/workspace/v2/desktops/results.go

```diff
@@ -130,6 +130,8 @@ type Desktop struct {
 	OrderId string `json:"order_id"`
 	// The key/value pairs of the desktop.
 	Tags []tags.ResourceTag `json:"tags"`
+	// EnterpriseProject ID of desktop
+	EnterpriseProjectId string `json:"enterprise_project_id,omitempty"`
```

Comment:
the omitempty is not needed in result, should be deleted

Reply:
Deleted